ext/TensorOperationsBumperExt.jl

```diff
@@ -7,7 +7,7 @@ function TensorOperations.tensoralloc(::Type{A}, structure, ::Val{istemp},
                                       buf::Union{SlabBuffer,AllocBuffer}) where {A<:AbstractArray,
                                                                                  istemp}
     # TODO: remove the `ndims` check if this is fixed in Bumper / StrideArraysCore
-    if istemp & ndims(A) > 0
+    if istemp & (ndims(A) > 0)
```

Comment:
I think the bug is that it should be `&&`, then the parentheses would not be necessary.